pyproject.toml

```diff
@@ -27,7 +27,7 @@ requests = "^2.31.0"
 tiktoken = "0.4.0"
 beautifulsoup4 = "^4.12.2"
 openai = "^0.27.8"
-langchain = "^0.0.201"
+langchain >= 0.0.220, < 0.1.0
```

Comment:
```suggestion
langchain = ">=0.0.201,<0.1.0"
```

Reply:
@bjornjorgensen Thanks for the fix. It requires quoting here.

Reply:
ok, I am committing the suggestion now. Let me know if you are against it.

Reply:
Yes, it works.

Reply:
Can this have anything to do with pyspark-ai https://github.com/langchain-ai/langchain#breaking-changes-for-select-chains-sqldatabase-on-728

Reply:
Not sure about that. Feel free to create a PR to increase the minimal required langchain version if you think it is necessary.